http/wsserver.go

```diff
@@ -66,6 +66,7 @@ func (s *WSServer) serveMessages(w http.ResponseWriter, r *auth.AuthenticatedReq
 	if host == "" {
 		host = r.RemoteAddr
 	}
+	logging.GetLogger().Debugf("serveMessages for client host %s for pool %s", host, s.GetName())
```

Comment:
"Serving messages for client %s for pool %s"

Reply:
fixed